assets/views/home/references.ejs

```diff
@@ -405,23 +196,7 @@
     </tr>
     <tbody id="collapseMisc" class="collapse">
     <tr ng-repeat="reference in <%= theUser %>.references | filter: isMisc | orderBy: 'createdAt'">
-      <td><strong>{{$index + 1}}.</strong></td>
-      <td colspan="2"><a ng-href="{{reference.url}}">{{reference.description || reference.descrip}}</a></td>
-      <td class="hidden-sm hidden-xs"></td>
-      <td>
-        <a href="#viewReferenceModal" data-toggle="modal" ng-click="editReference(reference)" class="icon">
-          <span class="glyphicon glyphicon-info-sign"></span>
-        </a>
-        <a href="#editReferenceModal" data-toggle="modal" ng-click="editReference(reference)"
-           ng-if="(!refUser || (user.name === refUser.name))" class="icon">
-          <span class="glyphicon glyphicon-edit"></span>
-        </a>
-        <a href="#"  ng-really-switch="deleteRef" class="icon"
-           ng-really-click="deleteRef(reference.id)"
-           ng-if="(!refUser || (user.name === refUser.name) || user.isMod)">
-          <span class="glyphicon glyphicon-trash"></span>
-        </a>
-      </td>
+      <%- partial('row.ejs') %>
```

Comment:
\o/ Now all we need is to collapse it more and have an ng-repeat for the sections as well. But that's a great start.